src/ontology/cl-edit.owl

```diff
@@ -31393,8 +31393,12 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:33351116") obo:IAO_00001
 AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033018 <https://orcid.org/0000-0001-6677-8489>)
 AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033018 <https://orcid.org/0000-0002-9185-3994>)
 AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033018 "2023-03-16T11:31:23Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:36524131") oboInOwl:hasExactSynonym obo:CL_4033018 "lung resident megakaryocyte")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:36524131") oboInOwl:hasExactSynonym obo:CL_4033018 "lung-resident megakaryocyte")
```

Comment:
Based on the definitions, one might expect 'lung(-) resident megakaryocyte' to be the label.

Reply:
Thank you @bvarner-ebi for the comment. I will ask the requester what is the preferred term.